code/style.css

```diff
@@ -1,4 +1,213 @@
+
+/* Universal styles */
+
 html {
-  background: #0025ff;
-  color: #fff;
+  box-sizing: border-box;
+  font-family: 'Work sans', Helvetica, sans-serif;
+  font-size: 16px;
+}
+
+body {
+  margin: 0;
+  background-color: #9DC88D;
+}
+
+h1, h2 {
+  font-family: 'Neuton', Garamond, serif;
+}
+
+p {
+  font-family: 'Work sans', Helvetica, sans-serif;
+  line-height: 1.4;
+}
+
+a {
+  font-family: 'Work sans', Helvetica, sans-serif;
```

Comment:
The same font-family is added/styled for the HTML in row 6 so I think this one can be removed as well.

Reply:
You are right, I'll remove the font-family.